### PR TITLE
Updated the description on the homepage

### PR DESCRIPTION
**Pull request by LAMike310**
Suggested by Rybofy in the #dev-marketing channel... I think it will help with the Facebook website previews

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -3,7 +3,7 @@
 <head>
 	<meta charset="utf-8"/>
 	<meta name="author" content="RockSteady" />
-	<meta name="description" content="TurtleCoin">
+	<meta name="description" content="TurtleCoin is a fun, fast, and easy way to send money to friends and businesses.">
```
Comment (ac1984) [inline index.html]: @LAMike310 How is this different from L9?

I've create an issue such things here: https://github.com/turtlecoin/turtlecoin.lol/issues/17. Feel free to tackle it :). @adj-gh did some of it here – https://github.com/turtlecoin/turtlecoin.lol/blob/master/brand.html

Comment (rybofy) [reply index.html]: Update looks great!It has turtleturtle.org in the title tho, is there a purpose for that URL? Also I would recommend adding a better image, maybe the logo..

image
@LAMike310

Comment (ac1984) [reply index.html]: @rybofy Re: turtleturtle.org – Yep, we've fixed that yesterday. @RocksteadyTC needs to deploy.

Comment (rybofy) [reply index.html]: Awesome thanks @ac1984

Comment (ac1984) [reply index.html]: @rybofy Re: the icons. Thats for bringing that to our attention. I've added it here: https://github.com/turtlecoin/turtlecoin.lol/issues/17.

**Pull request closed without merging**